### ISSUE TITLE
Regression: Segfault / No geometrical entities found on rebar (example file)'

### ISSUE THREAD

**Issue by johanrd** [IfcOpenShell/IfcOpenShell#459]
1. Experience abort (134) when converting [459--rebar--abort--augmented.ifc](https://github.com/IfcOpenShell/files/blob/master/459--rebar--abort--augmented.ifc) with c2f4c2c on OCCT 7.3.0:
```
*** Error in `IfcConvert_2x3': free(): invalid next size (fast): 0x00007fdadbc029c0 ***
FATAL: An error occured, status code 134.
```
2. Experience no segfault when converting with OCCT 7.2.0:
<img width="250" alt="screen shot 2018-09-10 at 13 07 44" src="https://user-images.githubusercontent.com/4601554/45294093-89b58b00-b4fa-11e8-8302-d50ce2e29fbe.png">

**Comment by johanrd** [comment]
Abort also happens on this file: [459--reinforcingBar--abort--2.ifc](https://github.com/IfcOpenShell/files/blob/master/459--reinforcingBar--abort--2.ifc) (about similar geometry), both on OCCT 7.3.0 and OCCT 7.2.0.

**Comment by johanrd** [comment]
Update: When converting with the OCCT patch from 34b3ca307fba0e8afb7512bf65f92eb0c5808a07, the conversion is no longer aborted, however there is no geometry output:
```[Error] {x} Error in: NCollection_Array1::ChangeValue
Failed to convert:
#1054665=IfcSweptDiskSolid(#1054654,6.,$,0.,365.)
[Error] No geometrical entities found```

**Comment by johanrd** [comment]
Update: This now segfaults on f24944b (v0.6.0):

```FATAL: An error occurred, status code 139.```

**Comment by johanrd** [comment]
Closing, as this seems to be fixed by 16cafb9 🎉

**Comment by johanrd** [comment]
Reopening, as there is no geometry output when converting from 12e9f7b (OCCT 7.3.0p3). Source file is augmented with expected quantities from 33cbcc2.

**Comment by johanrd** [comment]
Closing, as this seems to be fixed at 32588c6 🎉
